lib/friends/introvert.rb

```diff
@@ -476,6 +459,38 @@ def elapsed_days
 
     private
 
+    # Filter activites by friend, location and hashtag
```

Comment:
Same typo (here and in a lot of places). Could you just do a global find-replace?